Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'rails', '~> 7.0.8'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+gem 'pg'
 
```

Comment:
| @abdozayan12 |
|-|
| ![image](https://github.com/abdozayan12/DataApp/assets/93039359/00e9ad0a-14a1-4e7d-bf4c-2847931f26b6) |

- [ ] It would be nice if you updated your database.yml file because you have installed the PostgreSQL but didn't setup in the app because you can see that your app is still connected to the SQLite 🤔 Kindly update your database.yml file to connect/setup the PostgreSQL in your app correct 👍